collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -49,20 +48,6 @@ public class FillBlankFormTest {
     public RuleChain copyFormChain = TestRuleChain.chain()
             .around(rule);
 
-    @Test
-    public void exitDialog_ShouldDisplaySaveAndIgnoreOptions() {
```

Comment:
This is a test from the regression package written by the QA team and represents TestCase6 , TestCase9. I think we shouldn't remove such tests (we can update them but not remove) even if they are duplicated because then the QA team can't track which test cases are automated.

Reply:
@dbemke @srujner and I talked about this, and they track which ones they've automated elsewhere, so we're good to move that coverage around once it has been. Please jump into correct me if I've misremembered that!

Reply:
Ok they might be able to track it somehow but when something changes in the app they might need to update their test cases and having tests like that (created by the QA team) it is possible but they won't be able to update other types of tests we create. I think that:
- those regression tests should belong to them
- the QA team shouldn't trust us that we convert their tests into something else (that they do not fully understand) and everything is fine
- we (developers) shouldn't touch those tests
- we don't even need to run those tests every day. If it's a regression package maybe we can run it once a month for example and before releasing a new version.

Reply:
All good points. I'd not really thought about the regressions tests that way (although it is described like that in `TEST-GUIDELINES.md`), and we probably need to have a conversation about what we're doing with them. For now, I'll revert this change 👍.